web/src/components/RasaWidget.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement, useEffect } from 'react'
+import { useTheme } from 'styled-components'
+
+const RasaWidget = (): ReactElement => {
+  const theme = useTheme()
+
+  useEffect(() => {
```

Comment:
Can you please explain why we need the `useEffect` and `documentcreateElement` instead of just rendering it normally:
``` tsx
<script defer ... />
```

Reply:
This will load the script everytime the component is rendered.